exercise_02/callbacks/onError.js

```diff
@@ -7,3 +7,12 @@
  * output:
  *  - string: 'Error: user not created'
  */
+
+/**
+ * When called this function returns an error string
+ * @returns {string}
+ */
+function onError() {
+  return "Error: user not created";
+}
+export default onError;
```

Comment:
Improve Code Readability with Clearer Code Blocks. Fix it!
```suggestion

export default onError;
```